pkg/agent/monitortool/latency_store_test.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2024 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package monitortool
+
+import (
+	"context"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+var (
+	conn = &Connection{
+		ToIP:        net.ParseIP("10.244.2.1"),
+		Latency:     1 * time.Second,
+		Status:      true,
+		LastUpdated: time.Date(2024, 1, 1, 0, 0, 0, 0, time.UTC),
+		CreatedAt:   time.Date(2024, 1, 1, 0, 0, 0, 0, time.UTC),
+	}
+	conn2 = &Connection{
+		ToIP:        net.ParseIP("10.244.2.1"),
+		Latency:     2 * time.Second,
+		Status:      true,
+		LastUpdated: time.Date(2024, 1, 1, 0, 0, 0, 0, time.UTC),
+		CreatedAt:   time.Date(2024, 1, 1, 0, 0, 0, 0, time.UTC),
+	}
+	connectionMap = map[string][]*Connection{
+		"node1": {conn},
+	}
+	nodeGW0Map = map[string][]net.IP{
+		"node1": {net.ParseIP("10.244.2.1")},
+	}
+)
+
+func TestNewLatencyStore(t *testing.T) {
```

Comment:
I don't see why we need such a test function, it's not testing anything as far as I can tell.

Reply:
I think what @heanlan means is that we did not check any state changes in LatencyStore after Node events

Reply:
Thanks, I check it in `assert.EventuallyWithT(t, func(c *assert.CollectT) {}`.